src/Control/Control.js

```diff
@@ -1516,7 +1516,7 @@ var Control = module.exports = kind(
 		var was = this.content;
 		this.content = content;
 
-		if (was != content) this.notify('content', was, content);
+		if (was !== content) this.notify('content', was, content);
```

Comment:
This is actually located in the deprecated `setContent`, which I didn't notice during the first pass. When using the generic setter, there is no such restriction. I still think it might be worth updating this guard, at least for the sake of consistency.